dev-docs/bidders/yieldone.md

```diff
@@ -2,21 +2,186 @@
 layout: bidder
 title: YIELDONE
 description: Prebid YIELDONE Bidder Adaptor
-pbjs: true
-pbs: true
 biddercode: yieldone
-userIds: identityLink
 media_types: banner, video
+userIds: identityLink
+pbjs: true
+pbs: true
+
 ---
 
 ### Note:
 
-THE YIELDONE adapter requires setup and approval from the YIELDONE team.
+THE YIELDONE adapter requires setup and approval from the YIELDONE team.<br/>
 Please reach out to your account team or y1s@platform-one.co.jp for more information.
 
+
 ### Bid Params
 
 {: .table .table-bordered .table-striped }
-| Name          | Scope    | Description      | Example | Type     |
-|---------------|----------|------------------|---------|----------|
-| `placementId` | required | The placement ID | `'36891'` | `string` |
+| Name            | Scope    | Description                      | Example      | Type              |
+|-----------------|----------|----------------------------------|--------------|-------------------|
+| `placementId`   | required | The placement ID                 | `"36891"`    | `string`          |
+| `playerSize`    | optional | Video Resolution Size<br/>(This field is mandatory if mediaTypes.video.playerSize is [1, 1]) | `[640, 360]` | `array of number` |
```

Comment:
Is there any further information on the player available, e.g.

- what does 1x1 trigger? A responsive player?
- is the actual playerSize setting obeyed or is the player still changing its size